.releaserc.json

```diff
@@ -35,7 +35,14 @@
         "@semantic-release/git",
         {
           "message": "ci: regenerate code for version ${nextRelease.version}",
-          "assets": ["version", "equinix-openapi-metal/src"]
+          "assets": [
+            "version",
+            "equinix-openapi-metal/src",
+            "equinix-openapi-metal/README.md",
+            "equinix-openapi-metal/build.gradle",
+            "equinix-openapi-metal/build.sbt",
+            "equinix-openapi-metal/pom.xml"
```

Comment:
You can see that these are the files that the bot committed in 58f7d6a2482ea697aae907ba23fe465348acf11e (because they were missed by the previous run of the release job)